moto/cloudformation/models.py

```diff
@@ -715,8 +716,6 @@ def update_stack(self, name, template, role_arn=None, parameters=None, tags=None
 
     def list_stack_resources(self, stack_name_or_id):
         stack = self.get_stack(stack_name_or_id)
-        if stack is None:
-            return None
```

Comment:
The coverage decrease is caused by removing this block of code, since the number of relevant lines decreased and the missed lines stayed the same that decreased the coverage little bit. I will add some tests to cover some of the missing statements.